### DISCUSSION TITLE
Feedback on the Linux Build

### DISCUSSION THREAD

**Discussion by pomdtr** [johnlindquist/kit#679] [Errors]
Hi, thanks a lot for releasing a linux build !

I tried it on Fedora 35 and run into some issues.

1. The `ctrl+:` does not always works (dependending on the frontmost app). Fedora use Wayland as it's compositor, and it looks like the issue is related to this. See https://github.com/Ulauncher/Ulauncher/wiki/Hotkey-In-Wayland and https://github.com/albertlauncher/albert/issues/309 for more details.
2. Kit does not create a unix websocket on my machine. I wanted to create a shortcut in the gnome settings to trigger `kit` using kar (in order to bypass the first issue), but this was a blocker.
3. The first time I did run the installer, it became unresponsive at first (and gnome suggested to kill the app).

**Comment by nbaleli-w** [comment]
mind explaining how to install kit on linux please ?

**Reply by pomdtr** [reply]
An AppImage is available here: https://github.com/johnlindquist/kitapp/releases
